src/pg-quote.vue

```diff
@@ -24,29 +24,3 @@
     name: 'PgQuote'
   };
 </script>
-<style lang="scss" scoped>
```

Comment:
Removing styles because they are overwritten most of the time. keeping module size down.

Reply:
:+1: